### ISSUE TITLE
1.4.0 UnknownCommandError locating mexpire script.

### ISSUE THREAD

**Issue by nomoon** [sorentwo/readthis#47]
Trying to access the cache in rails I get the following error: 
```
Readthis::UnknownCommandError: unknown command 'mexpire'
from .../readthis-1.4.0/lib/readthis/scripts.rb:47:in `rescue in load_script!'
```

It seems that the gem can't find the lua file?

**Comment by sorentwo** [comment]
Thanks for the report, you're absolutely right. Bonehead move on my part, fixed with cd9de26 and released with v1.4.1

**Comment by nomoon** [comment]
Thanks very much!
